Source/WebCore/cssjit/SelectorCompiler.cpp

```diff
@@ -4042,9 +4008,12 @@ inline void SelectorCodeGenerator::generateElementHasTagName(Assembler::JumpList
 
         if (selectorLocalName == lowercaseLocalName) {
             // Generate localName == element->localName().
-            LocalRegister constantRegister(m_registerAllocator);
-            m_assembler.move(Assembler::TrustedImmPtr(selectorLocalName.impl()), constantRegister);
-            failureCases.append(m_assembler.branchPtr(Assembler::NotEqual, Assembler::Address(qualifiedNameImpl, QualifiedName::QualifiedNameImpl::localNameMemoryOffset()), constantRegister));
+            if (nameToMatch.nodeName() == NodeName::Unknown)
```

Comment:
We should be consistent in using `== NodeName::Unknown` or `!= NodeName::Unknown`

Reply:
It seems that this is the only place using this condition in if statement for now.